subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/report/Report.kt

```diff
@@ -27,6 +28,9 @@ interface Report {
 
     fun finish(isFullTestSuite: Boolean, reportViewerUrl: HttpUrl)
 
+    fun getTests(): Try<List<SimpleRunTest>>
```

Comment:
It's the first small step.
1) Add all needed functions in Report
2) Replace all direct usages of ReportApi
3) Hide Report creation behind factories
4) Refactor Report if needed

Reply:
Looks like SimpleRunTest also from `:report-viewer`? do you have plans to abstract it?
If no maybe it's better to move all the stuff from `:instrumentation-test` into `:report-viewer`? 
It's strange that we're increasing complexity of a gradle plugin module